src/views/LimitedRestorationExtension.vue

```diff
@@ -250,11 +268,6 @@ export default class Restoration extends Vue {
       // set applicant orgPerson
       entitySnapshot.orgPersons = this.parseApplicantOrgPerson(applicant)
 
-      // verify that business is in Limited Restoration status
-      // (will throw on error)
-      await this.verifyLimitedRestorationStatus(entitySnapshot.businessInfo)
```

Comment:
Removing this call because the method doesn't do anything at present.  It's mostly commented out.